doc/api/permissions.md

```diff
@@ -584,6 +584,12 @@ There are constraints you need to know before using this system:
 * Using existing file descriptors via the `node:fs` module bypasses the
   Permission Model.
 
+#### Allowing all write operations
+
+When `--allow-fs-write=*` is permitted, it may inadvertently lead to invalidating
+the permission model because of unintended file access, like full write access
+to memory with `/proc/self/mem`.
```

Comment:
How does it apply in a Node.js application? I understand the motivation, but I can't see how this could be exposable. 

At this point I don't think we need a runtime warning for that. I think adding a doc warning is ok.

Reply:
Many projects and programs may recommend permitting all IO operations ([like an 'issue' with pm2](https://github.com/nodejs/node/issues/53447) where the author granted all read permissions to pm2). A doc warning might be noticed by some, but the immediate jump for libraries that need access to system files outside of the current directory might grant *all* IO permissions instead of properly checking which ones to grant, which was *very* common in Deno libraries ([example 1 - inside deno](https://github.com/denoland/deno_blog), [example 2 - slack install script](https://github.com/slackapi/deno-slack-hooks?tab=readme-ov-file#install-update-script-usage))

Reply:
FWIW, I think this section might be too specific. There are many paths that users should probably never grant (read or write) access to.

Also, `/proc/self/mem` is a super specific example, and `/proc` is only one of many problematic things. (And yet, many applications will want to access `/proc` because it's the only way to implement many monitoring mechanisms on Linux.)